test/Common/DurableClientBaseTests.cs

```diff
@@ -214,7 +214,7 @@ public void DurableClient_ExternalApp_CreateHttpManagementPayload_ThrowsExceptio
 
         private IDurableOrchestrationClient GetDurableClient(IOrchestrationServiceClient orchestrationServiceClientMockObject)
         {
-            var storageProvider = new DurabilityProvider("test", new Mock<IOrchestrationService>().Object, orchestrationServiceClientMockObject, "test");
+            var storageProvider = new DurabilityProvider("test", new Mock<IOrchestrationService>().Object, orchestrationServiceClientMockObject, "test", "test");
```

Comment:
I realize this was here from before, but this line seems to be repeated, without modification, in a lot of tests. I wonder if we should track some refactoring item to abstract this repeated scaffolding logic